.builder/action/aws-c-http-test.py

```diff
@@ -33,6 +32,8 @@ def run(self, env):
             # consumer
             os.chdir('../../aws-c-http')
 
+        # actions.append(['ctest', '--output-on-failure', '-R',
+        #                'localhost_integ_h2_*'])
```

Comment:
??
did you meant to add this
usually a bad practice to commit commented-out code

Reply:
forgot to remove the code for testing